src/node/transaction.h

```diff
@@ -6,10 +6,13 @@
 #define BITCOIN_NODE_TRANSACTION_H
 
 #include <attributes.h>
+#include <consensus/params.h>
```

Comment:
You don't need to include this. You can forward declare Consensus::Params:

```c++
namespace Consensus {
struct Params;
}
```

Reply:
Ah, thanks! I was naively only `grep`ing for `Consensus::Params;` to look how other headers would forward-declare this (concluding that they don't and including the header is the preferred way), which is obviously not how it works :grimacing: